### PR TITLE
Improve slider thumb

### PR DESCRIPTION
**Pull request by mueller-ma**
Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt [hunk 1]
```diff
@@ -399,14 +421,19 @@ class WidgetAdapter(
         inflater: LayoutInflater,
         parent: ViewGroup,
         private val connection: Connection,
-        colorMapper: ColorMapper
+        colorMapper: ColorMapper,
+        context: Context
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt]: This is redundant, can use `itemView.context`.

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt [hunk 2]
```diff
@@ -399,14 +421,19 @@ class WidgetAdapter(
...
     ) : LabeledItemBaseViewHolder(inflater, parent, R.layout.widgetlist_slideritem, connection, colorMapper),
         SeekBar.OnSeekBarChangeListener {
         private val seekBar: SeekBar = itemView.findViewById(R.id.seekbar)
         private var boundWidget: Widget? = null
 
         init {
             seekBar.setOnSeekBarChangeListener(this)
+            val date = SimpleDateFormat("MM-dd", Locale.US).format(Date())
+            if (date == "10-31") {
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt]:
```
val now = Date()
if (now.dayOfMonth == 31 && now.month == Month.OCTOBER) {
```

Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt]: These functions are deprecated, but I the recommended alternative.

**Comment by mueller-ma** [comment]
> I assume this is going to be temporary? ;-)

I'd like to keep this.

**Pull request merged**